### DISCUSSION TITLE
PostGIS data set and photo table

### DISCUSSION THREAD

**Discussion by CEmond06** [opengisch/QField#5329] [Q&A]
I have a data set and I have also created a photo table and would like to bring it into the QField App, do I need to create a Package for QField or can I bring it in just as a direct PostGIS data set?

**Comment by timbo-barnes** [comment]
You can just link to the postgis layers directly, you just need to ensure that your 'secrets' have been added so that the QField cloud can connect and ensure that your postgis database will accept connections from QField cloud.
